From cab9830e77229ddafa41f91ab5572ed5f5b6ee93 Mon Sep 17 00:00:00 2001
From: sooorajjj <sooorajjj@gmail.com>
Date: Thu, 9 Feb 2017 13:23:01 +0530
Subject: [PATCH] Fix : Loading vendor-specific properties!

---
 init/property_service.cpp | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index 6010b8e..706ad0d 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -503,10 +503,6 @@ void load_persist_props(void) {
     load_override_properties();
     /* Read persistent properties after all default values have been loaded. */
     load_persistent_properties();
-
-    /* vendor-specific properties
-     */
-    vendor_load_properties();
 }
 
 void load_recovery_id_prop() {
@@ -551,6 +547,9 @@ void load_system_props() {
     load_properties_from_file(PROP_PATH_SYSTEM_BUILD, NULL);
     load_properties_from_file(PROP_PATH_VENDOR_BUILD, NULL);
     load_properties_from_file(PROP_PATH_FACTORY, "ro.*");
+    /* vendor-specific properties */
+    vendor_load_properties();
+    
     load_recovery_id_prop();
 }
 
-- 
2.7.4

